Linux.md

```diff
@@ -102,4 +104,10 @@
 `$!!`
 
 ### <a name="findWordInAFile"></a> Find word in a file
-`grep <wordYouWantToFind> <fileName.txt>`
\ No newline at end of file
+`grep <wordYouWantToFind> <fileName.txt>`
+
+### <a name="PrettifyfyPrintYamlFiles"></a> Prettify print yaml files
+`python -c 'import yaml,pprint;pprint.pprint(yaml.load(open(<PATH_OF_YAML_FILE>.read(), Loader=yaml.FullLoader))'`
```

Comment:
Replace it with `python3 -c 'import yaml;print(yaml.safe_load(open("<pathToFile.yaml>")))'`

Reply:
Well, Jein. this will print yaml data, but not print them pretty 

> > Updated.
> > But this may not compatible with old systems.Nevertheless we are going Toward the future and not Backward, so that make sense:) just wanted to express why I uses `python` at the beginning.
> 
> Looks like you didn't read my 1st comment completely, it was not about python3 only. The shorter is better.

Done.